docs/index.md

```diff
@@ -15,1619 +15,69 @@ parent="smn_workw_machine"
-function in the template.
-
-```
-$ docker-machine inspect --format='{{json .Driver}}' dev-fusion
-{"Boot2DockerURL":"","CPUS":8,"CPUs":8,"CaCertPath":"/Users/hairyhenderson/.docker/machine/certs/ca.pem","DiskSize":20000,"IPAddress":"172.16.62.129","ISO":"/Users/hairyhenderson/.docker/machine/machines/dev-fusion/boot2docker-1.5.0-GH747.iso","MachineName":"dev-fusion","Memory":1024,"PrivateKeyPath":"/Users/hairyhenderson/.docker/machine/certs/ca-key.pem","SSHPort":22,"SSHUser":"docker","SwarmDiscovery":"","SwarmHost":"tcp://0.0.0.0:3376","SwarmMaster":false}
-```
-
-While this is usable, it's not very human-readable. For this reason, there is
-`prettyjson`:
-
-```
-$ docker-machine inspect --format='{{prettyjson .Driver}}' dev-fusion
-{
-    "Boot2DockerURL": "",
-    "CPUS": 8,
-    "CPUs": 8,
-    "CaCertPath": "/Users/hairyhenderson/.docker/machine/certs/ca.pem",
-    "DiskSize": 20000,
-    "IPAddress": "172.16.62.129",
-    "ISO": "/Users/hairyhenderson/.docker/machine/machines/dev-fusion/boot2docker-1.5.0-GH747.iso",
-    "MachineName": "dev-fusion",
-    "Memory": 1024,
-    "PrivateKeyPath": "/Users/hairyhenderson/.docker/machine/certs/ca-key.pem",
-    "SSHPort": 22,
-    "SSHUser": "docker",
-    "SwarmDiscovery": "",
-    "SwarmHost": "tcp://0.0.0.0:3376",
-    "SwarmMaster": false
-}
-```
-
-#### help
-
-Show help text.
-
-#### ip
-
-Get the IP address of one or more machines.
-
-```
-$ docker-machine ip dev
-192.168.99.104
-$ docker-machine ip dev dev2
-192.168.99.104
-192.168.99.105
-```
-
-#### kill
-
-Kill (abruptly force stop) a machine.
-
-```
-$ docker-machine ls
-NAME   ACTIVE   DRIVER       STATE     URL
-dev    *        virtualbox   Running   tcp://192.168.99.104:2376
-$ docker-machine kill dev
-$ docker-machine ls
-NAME   ACTIVE   DRIVER       STATE     URL
-dev    *        virtualbox   Stopped
-```
+export DOCKER_HOST="tcp://192.168.99.100:2376"
+export DOCKER_CERT_PATH="/Users/mary/.docker/machine/machines/my-docker-machine"
+export DOCKER_MACHINE_NAME="my-docker-machine"
 
-#### ls
+The `DOCKER_TLS_VERIFY` sets transport layer security (TLS) on communication
+between your machine and the `DOCKER_HOST` machine. TLS is a protocol to allow
+client/server applications to communicate over the Internet securely. The
+`DOCKER_CERT_PATH` is the path to the certificate.  
 
-```
-Usage: docker-machine ls [OPTIONS] [arg...]
+Once you've configured your shell, you can use the Docker Engine `docker`
+command line commands to download images and run containers. For example, to
+start an `nginx` host:
 
-List machines
+         $ docker run -d -p 8000:80 nginx
+         
+Of course, this is a simple, local workflow. Where Docker Machine really excels
+is in provisioning and configuring multiple hosts. These host can be on your
+local network or on a remote, cloud provider. Additionally, Docker Machine makes it easy
+to deploy other Docker platform software such as Docker Compose and Docker Swarm.
```

Comment:
Would this be a good place to link off to `get-started.md` and `get-started-cloud.md`?